### PR TITLE
Add webhook sign validating feature inter-768

### PR DESCRIPTION
**Pull request by Orkuncakilkaya**

### CODE REVIEW

File: src/Webhook/WebhookVerifier.php [hunk 1]
```diff
@@ -0,0 +1,29 @@
+<?php
+
+namespace Fingerprint\ServerAPI\Webhook;
+
+final class WebhookVerifier
+{
+    public static function checkHeader(string $header, string $data, string $secret): bool
```
Comment (TheUnderScorer) [inline src/Webhook/WebhookVerifier.php:7]: Let's rename this method to something like `IsValidWebhookSignature` to keep it consistent with other SDKs.

**Review by TheUnderScorer — COMMENTED** [review]
LGMT, I have just one small suggestion :)

File: docs/Webhook.md [hunk 2]
```diff
@@ -0,0 +1,16 @@
...
+> Fingerprint\ServerAPI\Webhook\WebhookVerifier::IsValidWebhookSignature(string $header, string $data, string $secret): bool
+
+Verifies the HMAC signature extracted from the "fpjs-event-signature" header of the incoming request. This is a part of the webhook signing process, which is available only for enterprise customers.
+If you wish to enable it, please [contact our support](https://fingerprint.com/support).
+
+### Required Parameters
+
+| Name       | Type       | Description                                                | Notes |
+|------------|------------|------------------------------------------------------------|-------|
+| **header** | **string** | Value of the "fpjs-event-signature" header.                |       |
```
Comment (ilfa) [inline docs/Webhook.md:14]: May be we can use ` symbol to make it more readable?
```suggestion
| **header** | **string** | Value of the `fpjs-event-signature` header.                |       |
```

**Review by ilfa — APPROVED** [review]

**Review by TheUnderScorer — APPROVED** [review]

File: src/Webhook/WebhookVerifier.php (hunk 1 above)
Comment (Orkuncakilkaya) [reply src/Webhook/WebhookVerifier.php]: Hey 👋 Since you approved it, I mark this as resolved.

**Pull request merged**